opl/hbi_utils.py

```diff
@@ -188,10 +188,26 @@ def gen_send_verify(args, status_data):
     if args.dry_run:
         producer = None
     else:
+        try:
```

Comment:
Again, please use if so we do not log `Creating SASL password-protected producer to {args.kafka_host}` everytime.